docs/mux-logging-service.md

```diff
@@ -73,12 +77,17 @@ There are several new Ansible parameters which can be used with the
 
 * `openshift_logging_use_mux` - `True`/`False` - default `False` - if
   `True`, create the mux Service, DeploymentConfig, Secrets, ConfigMap, etc.
-* `openshift_logging_use_mux_client` - `True`/`False` - default `False` - if
-  `True`, configure the Fluentd collectors running on each node to send the raw
-  logs to mux instead of directly to Elasticsearch
-* `openshift_logging_use_mux_service` - `True`/`False` - default `False` - if
-  `True`, expose the mux service external to the cluster, and configure the mux
-  fluentd to accept logs from outside the cluster
+- `openshift_logging_mux_client_mode`: Values - `minimal`, `maximal`.
+  Default is unset.  Setting this value will cause the Fluentd node agent to
```

Comment:
Can we clarify what the behavior is when default?  If unset it operations in max mode? min mode?